README.md

```diff
@@ -25,10 +25,17 @@ branches as to be available for different runtimes.
 Jack audio support
 ------------------
 
-Ardour is build without Jack Audio support because Jack doesn't work
-from inside Flatpak.
+You need to install the package `pipewire-jack-audio-connection-kit` or
+equivalent in your distro. This might mean that you have to uninstall
+JACK if you had it installed.
 
-When PipeWire-Jack works we can consider the option.
+Then, open Ardour and it will work with the Pipewire implementation of
+JACK. If it doesn't work out of the box, just prefix launching Ardour
+with `pw-jack`, like here:
```

Comment:
No. This does not work.

With Flatpak, Ardour is completely sandboxed and pw-jack will have no effect.

Reply:
I might have been tricked by the fact it did work. But actually it worked the same without the `pw-jack` prefix, so you're probably right.